samples/packaged/CopilotChat/WebApp/README.md

```diff
@@ -2,11 +2,7 @@
 
 ## Running the sample
 
-1. You will need an
-   [Azure Open AI Service key and endpoint](https://learn.microsoft.com/azure/cognitive-services/openai/quickstart)
-   for this sample
-2. Ensure the KernelHttpServer sample is already running at `http://localhost:7071`. If not, follow the steps 
-   to start it [here](../../dotnet/KernelHttpServer/README.md).
+1. Ensure the SKWebApi running at `https://localhost:40443`.d).
```

Comment:
We should have a similar README for getting the SKWebApi up and running if we don't already (and reference it here).

Reply:
pushing to a future PR